roboteq_msgs/msg/EncoderCounts.msg

```diff
@@ -0,0 +1,3 @@
+#This message holds the encoder counts from the left and right encoders of the wheels
+float64 left_encoder
```

Comment:
Actually is it possible for the roboteq to have more than two encoders? If so you should prob make this a float64 array, and define some order given the encoder positions.

Reply:
According to the data sheet this Roboteq only supports two motors and encoders. So for this node that should be fine. If we ever end up finishing SMC we would do something similar to this